tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -110,10 +110,29 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
             openshift_ops.check_service_status_on_pod(
                 self.ocp_client, gluster_pod, service, "active", state)
 
+    def _bring_down_node_and_wait_to_become_notready(self, hostname):
```

Comment:
Change function name to `_power_off_node_and_wait_node_to_be_not_ready`

Reply:
Done